### DISCUSSION TITLE
Is there a way for the recently updated function to work without the downloaded files?

### DISCUSSION THREAD

**Discussion by CynicalPlatapus** [Avnsx/fansly#122] [Q&A] [answered]
So i performed a download of a content creator, now i want to move those files to a different drive and rename them. As far as i understand it the next time i perform a download, the downloader will re-download all of those photos and videos because it cannot find the original files with the hashes in their names, it won't detect any duplicates and will therefore re-download them all. Is there a way to prevent it from doing this, perhaps have a notepad file with all the hashes contained within it that it can update after the download completes, and read it next time it goes to do one.

**Comment by Avnsx** [comment]
Moving them isn't a problem if you switch the download_directory option accordingly in the config file.

Yeah you shouldn't re-name them, that will break deduplication

**Reply by CynicalPlatapus** [reply]
That's my point though, i have the download directory set as a specific folder on specific drive, and then i need to move the files to a different folder on a different drive, and i re-title the videos to match the subject of the videos.

I was hoping that a different means of preventing duplication could be used, a small text file for example with the hashes would only be a few kilobytes surely.

**Reply by Avnsx** [reply, marked as answer]
No I don't want to add a database file, as most people will just delete content within the download folder but not within the file where the hashes are stored and then everything will get mixed up again.

The only way to improve deduplication would be to actually add the hashes as metadata to each content, but that is very complicated to do programming wise due to the sheer variety of file extensions (.jpg, .png etc.) that would need to be uniquely supported.

**Reply by CynicalPlatapus** [reply]
Alright failing that, what about the ability to download files uploaded since the date that the last download was commenced? A few apps i have used such as RipMe or DownloaderForReddit have that feature, it's quite useful for preventing the re-downloading of files.

For example i did a download today, if the content creator posted something tomorrow and i ran the download again, it would be set to only download stuff posted after today.

**Reply by Avnsx** [reply]
Fansly downloader is already doing that, just run it again and it'll know (due to hashing) which content you've previously downloaded already and which one was posted recently by the content creator and still needs to be downloaded.
